wards.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
+
+[[ "$1" ]] || { echo "Please specify the ChainLog Key in ASCII to inspect"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+KEY=$(seth --to-bytes32 "$(seth --from-ascii "$1")")
+TARGET=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$KEY")
+
+echo -e "Network: $(seth chain)"
+LIST=$(seth call "$CHANGELOG" 'list()(bytes32[])')
+for key in $(echo -e "$LIST" | sed "s/,/ /g")
+do
+        ADDRESS=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$key")
```

Comment:
Getting gas errors here because mainnet fee spiked. Might want to set the gas envvar to some high number in the script so it doesn't affect reads.

```haskell
bmcmichael@bmcx1e:~/makerdao/spells-goerli$ ./wards.sh "MCD_ESM"
seth-rpc: {"id":1,"jsonrpc":"2.0","method":"eth_call","params":[{"data":"0x21f8a7214d43445f45534d00000000000000000000000000000000000000000000000000","from":"0xC0dE1AB51082442f821bA273273B4F7d32521D8b","gas":"0x895440","maxFeePerGas":"0x2e90edd000","maxPriorityFeePerGas":"0x77359400","to":"0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"},"latest"]}
seth-rpc: error:   code       -32000
seth-rpc: error:   message    err: max fee per gas less than block base fee: address 0xabc..., maxFeePerGas: 200000000000 baseFee: 206037515885 (supplied gas 9000000)
```

Reply:
Haven't reproduced it but d0709f8 should fix it.
Let me know if that works or some more tweaks are needed.

Reply:
I've adopted `bc` to handle big number in 81201f8